python/pyspark/sql/session.py

```diff
@@ -230,11 +230,6 @@ def enableHiveSupport(self) -> "SparkSession.Builder":
             """
             return self.config("spark.sql.catalogImplementation", "hive")
 
-        def _sparkContext(self, sc: SparkContext) -> "SparkSession.Builder":
-            with self._lock:
-                self._sc = sc
-                return self
```

Comment:
We only have one Spark context running globally now. It doesn't make sense to make it able to set Spark context anymore.

Reply:
And this func is private